boards/riscv/it8xxx2_evb/doc/index.rst

```diff
@@ -60,34 +59,159 @@ currently supports the following hardware features:
+
+   .. image:: it8xxx2_evb_setup.jpg
+        :width: 600px
+        :align: center
+        :alt: ITE Download Board Connected
+
+   .. note:: Be careful during connection!
+    Use separate wires to connect I2C pins with pins on the it8xxx2_evb board.
+    Wiring connection is described in the table below.
+
+    +-------------+---------------+
+    |   J5        | it8xxx2_evb   |
+    |   Connector | J145 Connector|
+    +=============+===============+
+    |      2      |       7       |
+    +-------------+---------------+
+    |      3      |       9       |
+    +-------------+---------------+
+    |      4      |       10      |
+    +-------------+---------------+
+
+#. Connect UART0 port of the it8xxx2_evb board
+   to your host computer using the usb cable.
+
+   .. note:: Be careful during connection!
+    Use separate wires to connect USB serial with pins on the it8xxx2_evb board.
+    Wiring connection is described in the table below.
+
+    +-------------+---------------+
+    |   USB       |  it8xxx2_evb  |
+    |   Connector |  Connector    |
+    +=============+===============+
+    |      TX     |   UART0 5     |
+    +-------------+---------------+
+    |      RX     |   UART0 7     |
+    +-------------+---------------+
+    |      GND    |      J15      |
+    +-------------+---------------+
+
+#. Apply power to the board via a USB cable.
 
 Building
 ========
 
-Applications for the ``it8xxx2_evb`` board configuration can be built as usual
-(see :`Zephyr Getting Started Guide`_):
+#. Build :ref:`hello_world` application as you would normally do
+   (see :`Zephyr Getting Started Guide`_):.
 
 .. zephyr-app-commands::
    :board: it8xxx2_evb
+   :zephyr-app: samples/hello_world
    :goals: build
 
+#. The file :file:`zephyr.bin` will be created if the build system
+   can find the image generation tool.
+
 Flashing
 ========
 
 In order to upload the application to the device,
-you'll need our winflash tool and download board.
+you'll need our flash tool and download board.
+
+Windows
+--------
+
 You can get the winflash tool at here:`Download Link`_
 Use the winflash.exe tool to program a zephyr application
 to the it8xxx2 board flash.
 
+#. Open winflash tool and make sure the order you open the switch of DownLoad
+   Board is right. Configure your flash tool like below.
+
+    .. figure:: WinFlashTool_P2.png
+          :width: 600px
+          :align: center
+
+    .. figure:: WinFlashTool_P4.png
+          :width: 600px
+          :align: center
+
+#. Using winflash tool flash zephyr.bin into your ITE board.
+   First, click ``Load`` button and select your zephyr.bin file.
+   Second, click ``run`` to flash the iamge into board.
+
+    .. figure:: WinFlashTool_P3.png
+          :width: 600px
+          :align: center
+
+#. Until here, you have flashed your image into ITE board and
+   it will work if you turn on ITE board. We can use MobaXterm
+   to test result.
```

Comment:
Make this text match the Ubuntu instructions.  "Use a terminal program to verify flashing worked correctly. You should see "Hello World! it8xxx2_evb" sent by the board.  If you don't see this message, press the Reset button and the message should appear."

Reply:
This section is about flashing on windows, updated.